src/main.rs

```diff
@@ -102,6 +88,25 @@ Some useful options (to be used as `cargo fuzz run fuzz_target -- <options>`) in
  - `-only_ascii`: Only provide ASCII input
  - `-dict=<file>`: Use a keyword dictionary from specified file. See http://llvm.org/docs/LibFuzzer.html#dictionaries")
         )
+        .subcommand(fuzz_subcommand("cmin")
+             .about("Corpus minifier")
+             .arg(Arg::with_name("CORPUS")
+                  .help("directory with corpus to minify"))
+        )
+        .subcommand(fuzz_subcommand("tmin")
+             .about("Test case minifier")
+             .arg(Arg::with_name("runs").long("runs")
+                  .help("number of attempts to minimize we should make")
+                  .takes_value(true)
+                  .default_value("255")
```

Comment:
I've actually had pretty good success with only 100 cases when I fuzzed `toml`, so I think 200 or 300 should work fine, but I have no problems bumping this up!